drivers/serial/Kconfig.nrfx

```diff
@@ -0,0 +1,79 @@
+# Kconfig.nrfx - NRFX UART configuration
+#
+# Copyright (c) 2016 - 2018, Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig UART_NRFX
+	bool "nRF UART nrfx drivers"
+	select SERIAL_HAS_DRIVER
+	select SERIAL_SUPPORT_INTERRUPT
+	depends on SOC_FAMILY_NRF
+	help
+	  Enable support for nrfx UART drivers for nRF MCU series.
+	  Peripherals with the same instance ID cannot be used together,
+	  e.g. UART_0 and UARTE_0.
+
+if UART_NRFX
+
+choice
+	prompt "UART Port 0 Driver type"
+
+config UART_0_NRF_UART
+	bool "nRF UART 0"
+	depends on ((SOC_SERIES_NRF52X || SOC_SERIES_NRF51X) && (!SOC_NRF52810))
+    select NRF_UART_PERIPHERAL
+	help
+	  Enable nRF UART without EasyDMA on port 0.
+endchoice
+
+if UART_0_NRF_UART
+
+config UART_0_NRF_PARITY_BIT
+        bool "Parity bit enabled"
```

Comment:
"Enable parity bit", to be consistent with "Enable flow control".

Reply:
fixed